### DISCUSSION TITLE
Disable the authorization button for users?

### DISCUSSION THREAD

**Discussion by githubhabIT** [walinejs/waline#2838] [General]
Is there any way to disable the authorization button for users and leave it only for the administrator

**Comment by lizheming** [comment]
Do you want to hide the login button or hide the social login button？

**Reply by githubhabIT** [reply]
![image](https://github.com/user-attachments/assets/c83bbba7-95b9-467c-bfd3-8b8a9b89ab0e)

**Comment by githubhabIT** [comment]
I don't want users to see it.

**Comment by lizheming** [comment]
You can use `login: 'disable'` to hide it for users. https://waline.js.org/en/reference/client/props.html#login
If you want to show it for yourself, you can set `login` property with some condition. For example, You can set `login: enable` when url has some specific query, otherwise it should be `disable` value.
